### PR TITLE
Edit feature description

### PR DESCRIPTION
**Pull request by wenmogu**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
```
Comment (joanneong) [inline docs/UserGuide.adoc]: I may not have been clear enough, but `v2.0` is the sub-section for features which we do not intend to implement within the scope of CS2103. Please shift these features which we do intend to implement to the correct section (i.e. the general `Features` section)

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
...
+
+=== Add relationship
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
```
Comment (joanneong) [inline docs/UserGuide.adoc]: If you look at the examples currently in the User Guide, you can see that the description should be more of a succinct explanation of what the feature achieves. For example, for the command `delete`, the description is "Deletes the specified person from the address book.", and the usage of the command is explained using the command `FORMAT` in the subsequent line. 

So for consistency's sake, I think it is better to follow this style as well when we update the User Guide
 
P/S It is "a relationship", not "an" :P

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
...
+as seen in the last shown list. +
+Format: `Add Relationship NAME1 NAME2`
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Inconsistency in the description and the command `FORMAT`. Above, you mentioned that a user should enter the "indices of the two people in between whom the user wants to add a relationship", but here your format suggests otherwise. Please make sure that your descriptions align with each other

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
...
+
+=== Add named relationship between people
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list and the name of the relationship. +
+If the relationship between these two people is not in the address book, one new relationship will be created upon +
+entering this command and given a name. +
+The order of the names of people entered does not matter.
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Same problems plaguing this part as the ones in the previous segment: (i) a description that explains the usage of the command instead and (ii) inconsistency in the suggested usage

Furthermore, because of the duplicity in the functionalities of this command and the abovementioned command, I think you may want to consider collapsing the two into `Add named relationship between people` such that if no name for the relationship is given as an argument by the user, a new unnamed relationship will simply be created

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
...
+and the estimate (the estimate >= 0 and <= 1). +
+Format: `Add CEstimate INDEX NAMEOFATTRIBUTE ESTIMATE`
+
+=== Delete confidence estimate for an attribute of a person
+
+Enters the index of the person as shown in the last seen list and the name of the attribute +
+of which the confidence estimate is to be deleted. +
+Format: `Delete CEstimate INDEX NAMEOFATTRIBUTE`
+
 == FAQ
```
Comment (joanneong) [inline docs/UserGuide.adoc]: In general, for all 4 commands introduced above, it would be good if you can add in some examples of the usages, particularly since the commands are slightly complicated and require several arguments

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -227,6 +227,35 @@ Format: `exit`
...
+Enters the index of the person as shown in the last seen list, the name of the attribute +
+and the estimate (the estimate >= 0 and <= 1). +
+Format: `Add CEstimate INDEX NAMEOFATTRIBUTE ESTIMATE`
+
+=== Delete confidence estimate for an attribute of a person
+
+Enters the index of the person as shown in the last seen list and the name of the attribute +
+of which the confidence estimate is to be deleted. +
+Format: `Delete CEstimate INDEX NAMEOFATTRIBUTE`
+
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Remember to edit this section as well (for the descriptions)

**Review by joanneong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (Xenonym) [reply docs/UserGuide.adoc]: My interpretation of the course documents is that all features not yet implemented are marked as `Coming in v2.0` until they are implemented, after which you mark them as `Since [version number]`. Is this incorrect?

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -274,7 +274,8 @@ Format: `undo`
 
 [NOTE]
 ====
-Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit` and `clear`).
+Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit`, `removeTag +
+` and `clear`).
```
Comment (joanneong) [inline docs/UserGuide.adoc]: the last part should be "`removeTag` and `clear`" (i.e. the ` should follow the removeTag directly, and not be on a new line). This will affect the display in the User Guide (feel free to click `View` above to verify).

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-== Features (Coming in v2.0)
+== Features(Coming in v2.0)
```
Comment (joanneong) [inline docs/UserGuide.adoc:350]: Why did you remove my spacing :(

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+
+=== Add a relationship
+
+Adds a relationship between two persons. +
+Format: `addRelationship INDEXOFSOURCE INDEXOFDESTINATION DIRECTION`
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Suggestion: Change to `SOURCEINDEX` or `SOURCE_INDEX` instead (same for destination) to improve readability for the user

File: docs/UserGuide.adoc [hunk 10]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+
+****
+* Add a relationship between the person with `INDEXOFSOURCE` and `INDEXOFDESTINATION`, +
+with the direction of the relationship specified.
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Very strange line breaks. I know for code we are not supposed to go beyond 120 characters per line (?) but for documentation the line breaks will affect the display. So please go through all the places (throughout the documentation!) where you placed such line breaks and remove them, so that they show up nicely (the text will wrap automatically, don't worry)

File: docs/UserGuide.adoc [hunk 11]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
```
Comment (joanneong) [inline docs/UserGuide.adoc]: How about adding a point before this that goes, "All indexes refer to the indexes shown in the most recent listing."

So that the `SOURCE_INDEX` can be `SOURCE_INDEX` refers to the index of the person from whom the relationship is initiated (i.e. phrased similarly to `DESTINATION_INDEX` but with greater clarity).

File: docs/UserGuide.adoc [hunk 12]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
```
Comment (joanneong) [inline docs/UserGuide.adoc]: "which refers to"

"....can only be `directed` or `undirected`" (this is a user guide)

File: docs/UserGuide.adoc [hunk 13]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+* `NAME` referring to the name of the relationship *can only be alphanumeric*.
+* If the `DIRECTION` is `directed`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` matters.
+* If the `DIRECTION` is `undirected`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` does not matter.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Change this section as per the suggestions for `addRelationship` since it is pretty much the same

File: docs/UserGuide.adoc [hunk 14]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+=== Add a confidence estimate
+
+Adds a confidence estimate for an attribute of a person. +
+Format: `addConfidenceEstimate INDEX TYPEOFATTRIBUTE ESTIMATE`
+
+****
+* `INDEX` refers to the index of the person whose attribute the user wants to add a confidence estimate to +
+as shown in the most recent listing.
+* `TYPEOFATTRIBUTE` refers to the `type` of the attribute that the confidence estimate is added to.
+* The `type` of the attribute refers to `Address`, `Email`, `Name`, `Phone`, `Tags`, `Relationships`.
```
Comment (joanneong) [inline docs/UserGuide.adoc]: combine this line with the previous line for greater clarity, i.e. "`ATTRIBUTE_TYPE` refers to the `type` of attribute that the confidence estimate is added to (e.g. `Address`, `Email`, `Name`, `Phone`, `Tags`, `Relationships`)."

File: docs/UserGuide.adoc [hunk 15]
```diff
@@ -341,7 +342,100 @@ Format: `exit`
...
+Deletes a confidence estimate for an attribute of a person. +
+Format: `deleteConfidenceEstimate INDEX TYPEOFATTRIBUTE`
+
+****
+* `INDEX` refers to the index of the person whose attribute the user wants to delete the confidence estimate to +
+as shown in the most recent listing.
+* `TYPEOFATTRIBUTE` refers to the `type` of the attribute that the confidence estimate is deleted from.
+* The `type` of the attribute refers to `Address`, `Email`, `Name`, `Phone`, `Tags`, `Relationships`.
+****
+
```
Comment (joanneong) [inline docs/UserGuide.adoc]: Same as for `addConfidenceEstimate`

**Review by joanneong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (joanneong) [reply docs/UserGuide.adoc]: yup sorry, my bad

File: docs/UserGuide.adoc (hunk 8 above)
Comment (wenmogu) [reply docs/UserGuide.adoc:350]: ...没有啊。。我这里没有看到extra spacing啊。。

File: docs/UserGuide.adoc (hunk 7 above)
Comment (wenmogu) [reply docs/UserGuide.adoc]: sorry brainfarted

File: docs/UserGuide.adoc (hunk 8 above)
Comment (joanneong) [reply docs/UserGuide.adoc:350]: I mean I wanted a spacing to be there, but don't worry about it, I can add it back in my next update for the UserGuide

**Review by joanneong — APPROVED** [review]
